.gitattributes

```diff
@@ -3,3 +3,5 @@ reporter/src/test/assets/*-expected-NOTICE*	text eol=lf
 
 # Fix GitHub language statistics, see https://github.com/github/linguist#generated-code.
 reporter-web-app/public/index.html		linguist-generated=true
+
+reporter-web-app/**/*.js			text eol=lf
```

Comment:
Commit message nit: "correctly work with the Git setting"

Reply:
Still not "correctly".

Reply:
Oh, I thought you were complaining about the missing "the" only. Sorry, fixed.